source/github-helpers/get-default-branch.ts

```diff
@@ -27,11 +26,12 @@ async function fromDOM(): Promise<string | undefined> {
 	return extractCurrentBranchFromBranchPicker(element);
 }
 
-async function fromAPI(repository: RepositoryInfo): Promise<string> {
+async function fromAPI(repository: NameWithOwner): Promise<string> {
+	const [name, owner] = repository.split('/');
```

Comment:
owner is first. 😂

Reply:
<img width="1631" alt="image" src="https://github.com/user-attachments/assets/ec7c4513-1d76-4e5a-9b58-66a9d2d45fbd">